### ISSUE TITLE
Rpi stream images to PC for image recognition

### ISSUE THREAD

**Issue by Aoi-00** [EbenKouao/pi-camera-stream-flask#21]
Hi! I chanced upon this project while looking for ways to link my rpi ( & pi camera) to send images continuously to my PC. The PC will then run image recognition and processing (yolov5) on it for my project. 

However, most guides I came across shows the pi camera just streaming the feed to a website. Is it possible to send the images to a PC instead for processing? Is there a way I can edit your implementation for this use case? If not, are you able to provide any guidance or codes?

Thank you in advance and sorry for the trouble! Really cool project by the way!

**Comment by Nichevo** [comment]
What I did was to stream the images to a Dropbox account folder that is linked on my PC so then the images were immediately available.
The Dropbox api is fairly simple.

**Comment by Aoi-00** [comment]
Oh! Does this not mean your dropbox folder gets increasingly bigger in size as the stream goes on?

**Comment by Nichevo** [comment]
Yes, it will end up using dropbox space but I didn't find it too much of a problem. You can clear them out onto your PC to keep the dropbox not getting too full if that's a problem. If you are streaming large volumes of pictures it might not be ideal and you might be better creating a server connection.

Here is an example of the code which uploads to a dropbox account

import dropbox
import re

TOKEN = 'asdfafdafasf'

dbx = dropbox.Dropbox(TOKEN)
#print(dbx.users_get_current_account())

sfilename = 'dropboxuploadtest3.py'
spath = '/Users/PythonProjects/'
sfile = spath + sfilename

stopath = '/' + sfilename
with open(sfile, "rb") as f:
    dbx.files_upload(f.read(), stopath, mode=dropbox.files.WriteMode("overwrite"))

sstorefile = 'piclist.csv'
with open(spath + sstorefile,'a') as file:
    file.write(sfilename + ',')

stopath = '/' + sstorefile
sfile = spath + sstorefile
with open(sfile, "rb") as f:
    dbx.files_upload(f.read(), stopath, mode=dropbox.files.WriteMode("overwrite"))
